### DISCUSSION TITLE
三星Galaxy S24旗艦機聚焦AI功能 18日全球發表會亮相

### DISCUSSION THREAD

**Discussion by Patsy17** [1101-GS4719-C/Python-Community-GS4719-C#1829] [General]
三星手機將加入Galaxy AI的人工智慧應用，對應各類生成式，如語音通話即時語言翻譯、AI搜尋等。
之前有用過朋友的三星手機，發現其實他的相機拍出來的比iphone還要好，畫質也很棒，
介面也滿好操作的，
但是好像許多人用過iphone就回不去安卓手機了～（我也是其中ㄧ個回不去的xd）

資料來源：https://www.cna.com.tw/news/ait/202401030014.aspx?topic=3437

**Comment by Ryanlolry** [comment]
三星加入Galaxy AI的人工智慧應用將為其手機帶來更多新功能，包括即時語言翻譯和AI搜尋等。雖然三星手機在相機表現、畫質和操作介面上都表現出色，但iPhone用戶可能已經習慣了其獨特的操作系統，對於特定功能和生態系統有著特殊偏好。手機選擇因個人喜好和使用習慣而異，這也解釋了為什麼有些人傾向於iPhone，而有些人則更喜歡三星手機。
